controller/lab_case_controller.py

```diff
@@ -13,16 +14,48 @@ def register_lab_case(case: LabCase):
 class LabCaseControllerImpl(LabCaseController):
     def __init__(self, db: LabCaseDB):
         self.db = db
-        
           
     def register_lab_case(self, case: LabCase):
         if self.db.fetch(case.case_id) == None:
             self.db.save(case)
         else:
             self.db.update(case)
+    
+    def import_allele_table(self, case: LabCase, file: str):
+        file = "tests/assets/allele_table_example.csv"
+        
+        with open(file) as csv_file:
+            csv_reader = csv.reader(csv_file, delimiter=',')
+            lines = []
+            for row in csv_reader:
+                lines.append(row)
+        
+        for line in lines:
```

Comment:
We can remove this `for` and use `line[i] = line[i].strip()` and `if line[3] == "": line[3] = line[2]` in the `for` below.

Reply:
```
        for line in lines[1:]:
            name = line[0].strip()
            locus = line[1].strip()
            allele_1 = line[2].strip()
            allele_2 = line[3].strip if  line[3].strip != "" else allele_1 
            genotype = Genotype(locus, allele_1, allele_2)
```